### PR TITLE
Resolved Integrate version 2.0.0 BasicServices and ApplicationPattern

### LINKED ISSUE #227: Integrate version 2.0.0 BasicServices and ApplicationPattern
Currently Application Layer Topology has its own basic service and application pattern modules.
This needs to be replace by include the following npm package for both ApplicationPattern and BasicServices are available,
•             onf-core-model-ap+2.0.0
•             onf-core-model-ap-bs+2.0.0

Upgrade ApplicationPattern and BasicService npm package to version 2.0.0.
Changes ,

- [ ] In the file /server/package.json , change the versions of the following two packages to 2.0.0
image
- [ ] In the version 2 of the specification , the service redirect-topology-change-request consists of a responseBody. So the response code in the function redirectTopologyChangeInformation in the module server\controllers\BasicServices.js needs to be changed to responseCodeEnum.code.OK.
- [ ] In the version 2 of the specification , the service start-application-in-generic-representation moved to the BasicServices. So need to move the function startApplicationInGenericRepresentation from the file server\controllers\IndividualServices.js to server\controllers\BasicServices.js
- [ ] Remove the file server\service\BasicServicesService.js and in the file server\controllers\BasicServices.js , include the following import to direct the function calls to the npm package , 
var BasicServices = require('onf-core-model-ap-bs/basicServices/BasicServicesService');
- [ ] Remove the folders server/applicationPattern and server/basicServices
- [ ] Perform basic test and validate the changes and push the code.

### PR DESCRIPTION
**Pull request by ManasaBM1**
Fixes #227

### CODE REVIEW

**Comment by ManasaBM1** [comment]
Reviewer is @vanithavalluripalli9

File: server/package.json [hunk 1]
```diff
@@ -21,6 +21,8 @@
     "node-statsd": "^0.1.1",
     "oas3-tools": "^2.2.3",
     "randexp": "^0.5.3",
-    "response-time": "^2.3.2"
+    "response-time": "^2.3.2",
+    "onf-core-model-ap-bs": "2.0.0-alpha.3", 
+    "onf-core-model-ap": "2.0.0-alpha.3"
```
Comment (DanaSunal) [inline server/package.json:26]: Please change it to latest as per openBackhaul/ExecutionAndTraceLog#195

Comment (ManasaBM1) [reply server/package.json:26]: Done

Comment (ManasaBM1) [reply server/package.json]: Done

File: server/package.json [hunk 2]
```diff
@@ -19,10 +19,10 @@
...
     "moment": "^2.29.1",
     "node-statsd": "^0.1.1",
+    "randexp": "^0.5.3",
+    "response-time": "^2.3.2",
     "onf-core-model-ap": "2.0.0-alpha.4",
     "onf-core-model-ap-bs": "2.0.0-alpha.4",
-    "openbackhaul-oas3-tools": "2.3.1-alpha.1",
-    "randexp": "^0.5.3",
-    "response-time": "^2.3.2"
+    "openbackhaul-oas3-tools": "2.3.1-alpha.1"
```
Comment (DanaSunal) [inline server/package.json:26]: packages should be in alphabetical order.

**Review by DanaSunal — CHANGES_REQUESTED** [review]
Some files from the inline applicationPattern are needed in IndividualServices, please check and keep them.

**Review by DanaSunal — APPROVED** [review]

**Pull request merged**